src/Site_WP_Docker.php

```diff
@@ -129,19 +129,37 @@ public function generate_docker_compose_yml( array $filters = [] ) {
 				'name' => 'io.easyengine.site=${VIRTUAL_HOST}',
 			],
 		];
-		$nginx['networks']    = [
-			'net' => [
-				[
-					'name' => 'site-network',
-					'aliases' => [
-						'alias' => [
-							'name' => '${VIRTUAL_HOST}',
+
+		if ( in_array( GLOBAL_REDIS, $filters, true ) ) {
+			$nginx['networks']    = [
+				'net' => [
+					[
+						'name' => 'site-network',
+						'aliases' => [
+							'alias' => [
+								'name' => '${VIRTUAL_HOST}',
+							],
 						],
 					],
-				],
-				[ 'name' => 'global-frontend-network' ],
-			]
-		];
+					[ 'name' => 'global-frontend-network' ],
+					[ 'name' => 'global-backend-network' ],
+				]
+			];
+		} else {
+			$nginx['networks']    = [
```

Comment:
This both blocks added in if/else are almost same. You can just add what you need in condition. If it's the whole block then it's ok but not here.

Something like this, 
```php
$nginx['networks']    = [
				'net' => [
					[
						'name' => 'site-network',
						'aliases' => [
							'alias' => [
								'name' => '${VIRTUAL_HOST}',
							],
						],
					],
					[ 'name' => 'global-frontend-network' ],
				]
			];

if ( in_array( GLOBAL_REDIS, $filters, true ) ) {
     $nginx['networks']['net'][] =  [ 'name' => 'global-backend-network' ];
}
```
P.s: correct it if array neasting is not proper. I just added this for ref.